### DISCUSSION TITLE
Welcome to openwrt-useful-tools Discussions!

### DISCUSSION THREAD

**Discussion by adde88** [adde88/openwrt-useful-tools#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. Feel free to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by piercejpierce** [comment]
How are you using this on WiFi pineapple mark 7 including installing?

**Comment by m0bilesecurity** [comment]
Hey @adde88,
are you planning to update bettercap?
Many thanks

**Reply by ChillVibesMushroom** [reply]
It’s been updated
